### ISSUE TITLE
Background Geo Location

### ISSUE THREAD

**Issue by rguntha** [ionic-team/capacitor#516]
Dear Friends At Capacitor,

We are working on several applications which need to track the geo location of moving vehicles.  We need a background geo location plug similar to 'https://www.npmjs.com/package/cordova-plugin-mauron85-background-geolocation' which works when the phone screen is off, app is pushed to background, and even when the app is closed.

Thanks for your consideration.

I am really excited about Capacitor.

**Comment by pufanalexandru** [comment]
Any updates on this? We want to migrate our app to Capacitor, but we need the background geolocation tracking for our app, and somehow we can't make cordova-plugin-mauron85-background-geolocation to build with Capacitor.

**Comment by danielkappelle** [comment]
Have you looked at https://ionicframework.com/docs/native/background-geolocation ?

**Comment by pufanalexandru** [comment]
> Have you looked at https://ionicframework.com/docs/native/background-geolocation ?

Yes, but as mentioned here, I can't make it work with Capacitor: https://stackoverflow.com/questions/61520250/cannot-build-ionic-app-on-android-once-installed-backgroundgeolocation-plugin-wi

**Comment by danielkappelle** [comment]
Ah yeah, I'm having some trouble still as well. Trying to get it to work... If it works, I'll comment below.
Alternatively I'll check out https://github.com/transistorsoft/cordova-background-geolocation-lt

**Comment by danielkappelle** [comment]
I've gotten it to build at least, I've commented it on stackoverflow.

**Comment by rasheeek** [comment]
Check out this plugin
https://github.com/capacitor-community/background-geolocation/tree/master
